src/ensembl/webpack/webpack.config.prod.js

```diff
@@ -103,7 +103,12 @@ const plugins = [
   new WorkboxPlugin.GenerateSW({
     swDest: '../service-worker.js', // save service worker in the root folder (/dist) instead of /dist/static
     clientsClaim: true,
-    skipWaiting: true
+    skipWaiting: true,
+    exclude: [/index.html$/, /\.gz$/, /\.br$/, /\.js\.map$/],
+    runtimeCaching: [{
```

Comment:
The only purpose of runtimeCaching is that we can open/reload our application when offline, and it will show something (very little, and likely broken, but still something rather than nothing). I am not quite sure whether it's useful.

If we remove runtimeCaching, our site will not show anything when offline, but all our static assets produced by webpack will still be requested and cached by service worker. Which is the most value out of service worker that we are currently getting anyway.

Reply:
If there is no harm in having this, why not? Or is there any disadvantages using this?